pkg/kubelet/eviction/helpers.go

```diff
@@ -1243,6 +1252,23 @@ func evictionMessage(resourceToReclaim v1.ResourceName, pod *v1.Pod, stats stats
 	if !ok {
 		return
 	}
+
+	// Pod level resources will be included in eviction message, along container resources
+	if utilfeature.DefaultFeatureGate.Enabled(features.PodLevelResources) && resourcehelper.IsPodLevelResourcesSet(pod) && resourcehelper.IsSupportedPodLevelResource(resourceToReclaim) {
+		podRequest := v1resource.GetResourceRequestQuantity(pod, resourceToReclaim)
+
+		var podUsage *resource.Quantity
+
+		if resourceToReclaim == v1.ResourceMemory && podStats.Memory != nil {
```

Comment:
Is memory the only reclaimable pod-level resource? In that case, check `resourceToReclaim == v1.ResourceMemory` in the previous if-condition (line 1257)